apps/web/.env.production

```
@@ -1,20 +1,26 @@
+
+# .env.production
 # ============================================
 # 本番環境用設定（共有設定）
-# このファイルはgit管理されます
-# 実際の本番環境ではVercelの環境変数で上書きされます
+# このファイルはdotenvxで暗号化されてgit管理されます
 # ============================================
+# 環境設定
+NEXT_PUBLIC_ENVIRONMENT=encrypted:BErPsd4aKdiGhJlohotmQaSXbj7WcOhd+ezwUn4Rj7ZUoZDecG2rPjV7GU+dW/B4hy7OreeE11pojqRzU42VM/l3lJGWO6wmpnu8HdOk/Au+PKGdGVIu37gd0uMNWnOuuT4fA63eUeOWZpk=
+# アプリケーションURL（本番環境）
+NEXT_PUBLIC_APP_URL=encrypted:BLsowhz62AQ0n+nJCBpAvPSZVULbpPfXABeCy3jhUuk6mXzvALTTNCe8EqaGqvLOkZNGsUIDRXRb09oKkX7QHUZz6ShwjC2tFWgAsvQkwH3X7HVuDGYTMOWgHKMOcrPuEYyRiBjRkY5feWkj/0uXw58RUsP8eolppyq5jA==
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_APP_URL key should go before the NEXT_PUBLIC_ENVIRONMENT key

(UnorderedKey)